frontend/src/components/Pages/RobotPage/RobotArmMovement.tsx

```diff
@@ -35,7 +55,9 @@ export function MoveRobotArm({ robot, armPosition }: RobotProps) {
     }
     return (
         <>
-            <Button onClick={onClickMoveArm}>{TranslateText('Set robot arm to ') + '"' + armPosition + '"'}</Button>
+            <Button style={moveArmButtonStyle()} disabled={!usable} onClick={onClickMoveArm}>
```

Comment:
```suggestion
            <Button disabled={!usable} onClick={onClickMoveArm}>
```
No need with custom disabled style. Happens automatically when disabled

Reply:
If we don't use the built in disable I would use this style: 
"
style = {{
    backgroundColor: tokens.colors.interactive.disabled__fill.hex,
    color: tokens.colors.interactive.disabled__text.hex,
    borderColor: tokens.colors.interactive.disabled__border.hex,
    cursor: "not-allowed",
}}"

to make sure that the style is the same as other disabled buttons

Reply:
done